### PR TITLE
remove game objects from synchvars

### PR DESCRIPTION
**Pull request by Bod9001**
this is due to mirror doing a get component in the multithreaded portion, and causing unity to explode because it's apparently impossible to do a get component from a different thread
### Notes:

CL: [Fix] should fixed weird synchronisation issues

### CODE REVIEW

File: UnityProject/Assets/Scripts/Managers/NetworkManagement/CustomNetworkManager.cs [hunk 1]
```diff
@@ -45,6 +45,9 @@ public class CustomNetworkManager : NetworkManager, IInitialise
 	/// </summary>
 	[NonSerialized] public UnityEvent OnClientDisconnected = new UnityEvent();
 
+	public static Dictionary<uint, NetworkIdentity> spawned => IsServer ? NetworkServer.spawned : NetworkClient.spawned;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Managers/NetworkManagement/CustomNetworkManager.cs]:
```suggestion
	public static Dictionary<uint, NetworkIdentity> Spawned => IsServer ? NetworkServer.spawned : NetworkClient.spawned;
```

File: UnityProject/Assets/Scripts/Items/Bureaucracy/PaperBin.cs [hunk 2]
```diff
@@ -16,7 +16,13 @@ public class PaperBin : NetworkBehaviour, ICheckedInteractable<HandApply>, IServ
 		private int paperCount;
 
 		[SyncVar (hook = nameof(SyncStoredPen))]
-		private GameObject storedPen;
+		private NetworkIdentity _storedPenID;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Bureaucracy/PaperBin.cs:19]: Why the underscore?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Items/Bureaucracy/PaperBin.cs:19]: Ryder suggested it

Comment (corp-0) [reply UnityProject/Assets/Scripts/Items/Bureaucracy/PaperBin.cs:19]: It probably was copilot. Underscore for privates isn't our code style.

**Pull request merged**